specs/resources/authorization.yml

```diff
@@ -0,0 +1,59 @@
+post:
+  tags:
+    - token
+  summary: oauth
+  description: OAuth get token response
```

Comment:
Suggested edit: "Generates an access token that contains permissions scoped to the application owner's Dwolla Client Account."